receiver/redisreceiver/metadata.yaml

```diff
@@ -284,29 +285,29 @@ metrics:
   redis.replication.backlog_first_byte_offset:
     enabled: true
     description: The master offset of the replication backlog buffer
-    unit: ""
+    unit: "By"
     gauge:
       value_type: int
 
   redis.replication.offset:
     enabled: true
     description: The server's current replication offset
-    unit: ""
+    unit: "By"
```

Comment:
These also are breaking changes. Will you open a dedicated PR for these?

Reply:
Also added in other PR: #23454